README.md

```diff
@@ -1 +1,42 @@
-# mon
+# Mon
+
+Mon is a lightweight Haskell library used for monitoring (client). Purpose of the library is to achieve monitoring functionality by adding minimal weight on the code being monitored e.g. small set of dependencies and minimal code integration. This is accomplished with simple `MonadIO` functions that send metrics to the receiving server.
```

Comment:
I mean, that's probably right. But I think you'd be better off stating that the purpose of the library is to make it easy and typesafe to emit metrics using the StatsD protocol.